models/zeroex/base/zeroex_base_api_fills_schema.yml

```diff
@@ -0,0 +1,133 @@
+version: 2
+
+models:
+  - name: zeroex_base_api_fills
+    meta:
+      blockchain: base
+      project: zeroex
+      contributors: rantum
+    config:
+      tags: ["base","0x","dex_aggregator","dex","aggregator"]
+    description: >
+        0x API erc20 swaps raw fills (including multihops) and also native swaps thru 0x exchange contracts (without using 0x API) on base
+    tests:
+      - dbt_utils.unique_combination_of_columns:
+          combination_of_columns:
+            - block_time
+            - tx_hash
+            - evt_index
+            - taker_token
```

Comment:
looks like this test needs updated too, to match unique keys in model

Reply:
Could you clarify?

Reply:
you have more columns in the unique combo of columns test than you do actual unique keys in the model config itself